ganttproject/src/biz/ganttproject/impex/google/GoogleExportOptionPageProvider.java

```diff
@@ -0,0 +1,55 @@
+package biz.ganttproject.impex.google;
+
+import biz.ganttproject.core.option.GPOptionGroup;
+import net.sourceforge.ganttproject.gui.options.OptionPageProviderBase;
+import net.sourceforge.ganttproject.gui.options.OptionsPageBuilder;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.io.IOException;
+
+public class GoogleExportOptionPageProvider extends OptionPageProviderBase {
```

Comment:
This class should be registered in `plugin.xml` as extension. Did you forget to add `plugin.xml` to the pull request?